touchforms/backend/persistence.py

```diff
@@ -24,18 +25,18 @@ def restore(sess_id, factory, override_state=None):
         state.update(override_state)
     return factory(**state)
 
-# TODO integrate with real caching framework (django ideally)
-def cache_set(key, value, timeout):
-    with open(cache_path(key), 'w') as f:
-        f.write(json.dumps(value).encode('utf8'))
 
-# TODO integrate with real caching framework (django ideally)
+def cache_set(key, value):
+    if key is None:
+        raise KeyError
+    postgres_insert(key, value)
+
+
 def cache_get(key):
-    try:
-        with open(cache_path(key)) as f:
-            return json.loads(f.read().decode('utf8'))
-    except IOError:
+    if key is None:
         raise KeyError
+    try:
+        return postgres_lookup(key)
     except JSONDecodeError:
         raise EmptyCacheFileException(_(
             u"Unfortunately an error has occurred on the server and your form cannot be saved. "
```

Comment:
we should fall back to the old lookup for now. i think that's a better plan than doing a migration before deploying it